libs/common/src/log_level.cpp

```diff
@@ -11,13 +11,12 @@ char const* GetLogLevelName(LogLevel level, char const* default_) {
             return "debug";
         case LogLevel::kInfo:
             return "info";
-        case LogLevel::kWarn: {
```

Comment:
This pair of braces was here.. must've been a typo. It shouldn't have caused any issue, but it was breaking the indentation.